boringtun/src/noise/mod.rs

```
@@ -959,6 +962,54 @@ mod tests {
         update_timer_results_in_handshake(&mut my_tun, &mut now);
     }
 
+    /// If a tunnel is idle for close to 120s without sending a packet,
```

What do we use for "close to", one second? This seems quite arbitrary, no?

I'm curious how the spec handles this edge case. Why not just let the session fail?

---

See: https://github.com/firezone/boringtun/pull/68/files#diff-9aeaf76bd37d0a89c78123a398c4e238c9a74a2a638c90ee0c57e82e0c315b60R30-R31

Defined as 10s as per definition of `KEEPALIVE_TIMEOUT`.